src/main/webapp/style.css

```diff
@@ -2,6 +2,11 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300;400;500&display=swap');
 
 /* General CSS styles that could apply across all of our pages */
+*{
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+}
```

Comment:
These styles seemed to shrink a lot of the website when I ran a local development branch since they're applied to all elements and remove default paddings and margins. Can we move them to more specific elements instead? I moved stuff around and found that these produce the same effect:
```
body {
    margin: 0px; /* So that the navbar takes up the full width. */
}

.nav {
height: 65px /* Margin changed the view height so this keeps navbar constant */
/* Remove min-height property */
}

.nav-links {
padding: 0px; /* Gives space between nav links by removing ul's padding */
}
```

And then after these changes I found that the `box-sizing: border-box;` had no effect anymore, so I could remove it.
Let me know what you think and if it produces the same result for you.